tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -4,11 +4,14 @@ enum XCSchemeConstants {
     // GH399: Derive the defaultLastUpgradeVersion
     static let defaultLastUpgradeVersion = "1320"
     static let lldbInitVersion = "1.7"
-    static let defaultBuildConfigurationName = "Debug"
     static let posixSpawnLauncher = "Xcode.IDEFoundation.Launcher.PosixSpawn"
     static let customLLDBInitFile = "$(BAZEL_LLDB_INIT)"
 }
 
+extension String {
+    static let defaultBuildConfigurationName = "Debug"
```

Comment:
I prefer refactor and behavior changes to be in separate PRs. Can you make this change in its own PR?

Reply:
#804